site/en/docs/privacy-sandbox/fledge-best-practices/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'FLEDGE latency best practices'
+subhead: >
+  How to get the best performance from FLEDGE usage
+description: >
+  Collection of best practices to improve FLEDGE auction latency
+date: 2022-06-27
+authors:
+  - pauljensen
+---
+
+<!--lint disable no-smart-quotes-->
+
+It's in everyone's best interest to make sure FLEDGE operates efficiently.
+
+* People browsing the web want sites to load quickly. This means FLEDGE must not overutilize limited device resources, like compute or network resources, necessary to load sites and their embedded ads.
```

Comment:
Is it up to FLEDGE to not overuse resources? Or is it up to the site developer to build with FLEDGE efficiently, as to not overuse limited device resources?

Reply:
In the context of this article, it is up to the site developer to setup FLEDGE in an efficient way, since FLEDGE can be heavy in resources.

Reply:
```suggestion
* People browsing the web want sites to load quickly. This means developers should build with FLEDGE efficiently as to not overutilize limited device resources, like compute or network resources, that are necessary to load sites and their embedded ads.
```

Reply:
This is a good suggestion, since the article is talking about best practice for the use of FLEDGE, not the implementation/design of FLEDGE.